lib/host/aws.js

```diff
@@ -92,7 +94,13 @@ function* configure (baseUrl) {
   return config;
 };
 
+function* getSpotTerminationTime() {
```

Comment:
naming nit: We have generic names in this file for getTerminationTime is probably better then including spot in the name here.